src/vlx/README.md

```diff
@@ -0,0 +1,76 @@
+# @vltpkg/vlx
+
+Executable management for `vlt exec`.
+
+**[Usage](#usage)**
+
+## Overview
+
+This is a tool that provides the global installs used by `vlt exec`
+(that is, when it's not a local installed dependency).
+
+## Usage
+
+```js
+import * as vlx from '@vltpkg/vlx'
+
+// get the options and arguments in the normal ways
+const conf: LoadedConfig = blahblahblah()
+
+const { positionals, options} = conf
+
+// Provide a promptFn which is ()=>Promise<string> if you want
```

Comment:
```suggestion
// Provide a promptFn which is `()=>Promise<string>` if you want
```

This is what's causing the docs CI job to fail. Since this all gets parsed as MDX eventually we have to always do MDX type things even in markdown.